website/docs/installation/Optional-Dependencies.md

```diff
@@ -1,5 +1,16 @@
 # Optional Dependencies
 
+## Different LLMs
+
+AutoGen installs OpenAI package by default. To use LLMs by other providers, you can install the following packages:
+
+```bash
+pip install pyautogen[gemini,anthropic,mistral,together,groq,cohere]
+```
+
+Check out the [notebook](https://github.com/autogen-ai/autogen/blob/main/notebook/autogen_uniformed_api_calling.ipynb) and
+[blogpost](../../blog/2024/06/24/AltModels-Classes) for more details.
```

Comment:
Sorry I put the comment in the resolved thread so it is not shown.

I am changing the notebook link back to URL since it is not compiled in the website.

Reply:
My mistake, the reference should work. @sonichi I have fixed the broken urls. Can you take a look now?